### PR TITLE
Modify view

### PR DESCRIPTION
**Pull request by nazliuzuner**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sweng/thegramginers/UserProjects/UserProjectsFragment.java [hunk 1]
```diff
@@ -60,6 +61,11 @@ public void onViewCreated(View view, @android.support.annotation.Nullable Bundle
             public void onNewElements(List<Project> newElements, boolean isRefreshed) {
                 if(isRefreshed) userProjectList.clear();
                 userProjectList.addAll(newElements);
+
+                if(userProjectList.isEmpty()){
+                    Toast.makeText(getContext(), "No projects found", Toast.LENGTH_LONG).show();
```
Comment (KnightEther) [inline app/src/main/java/ch/epfl/sweng/thegramginers/UserProjects/UserProjectsFragment.java]: You will have UI threads issues with this toast
You should put this code in updateUI inside the runOnUIThread

Comment (KnightEther) [reply app/src/main/java/ch/epfl/sweng/thegramginers/UserProjects/UserProjectsFragment.java]: This concerns as well other fragment with the same functionnality

Comment (nazliuzuner) [reply app/src/main/java/ch/epfl/sweng/thegramginers/UserProjects/UserProjectsFragment.java]: Thank you for noticing ! The changes you requested are made on my last commit

**Comment by KnightEther** [comment]
This PR will be merge with "Merge PRs" PR

**Pull request closed without merging**